integration_tests/e2eReferDisabled/refer.cy.ts

```diff
@@ -30,6 +30,22 @@ context('Refer', () => {
     notFoundPage.shouldContain404H2()
   })
 
+  it("Doesn't show the new page for a referral", () => {
+    cy.signIn()
+
+    const course = courseFactory.build()
+    const courseOffering = courseOfferingFactory.build({})
+
+    cy.task('stubCourse', course)
+    cy.task('stubCourseOffering', { courseId: course.id, courseOffering })
```

Comment:
Not sure if we need these - might be good to audit these stubs in another card to check which are needed

Reply:
We don't need these, as we're not (in theory) looking anything up. They're useful in the test above as we wouldn't know if the page was failing to load due to an error fetching data (with the stubs not being set up properly) or due to the whole journey being disabled.

We can remove them here, though.

Agree, it would be good to have a quick audit of stubs across other tests though.

Reply:
Yep, that's right